build.gradle

```diff
@@ -31,8 +31,8 @@ dependencies {
     modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
     // fabric api resource loader
-    modCompile "net.fabricmc.fabric-api:fabric-resource-loader-v0:${project.fabric_resource_loader_version}"
-    include "net.fabricmc.fabric-api:fabric-resource-loader-v0:${project.fabric_resource_loader_version}"
+    modCompile "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
+    include "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
```

Comment:
Yeah, no.  I personally don't like the idea of including all of FAPI.  What other parts of it do you use?

Reply:
It's implementing only

Reply:
to answer your question: resource loader and registry sync
jijing registry sync is concern and its something nobody ever does